src/dlgVarsMainArea.cpp

```diff
@@ -71,12 +75,12 @@ dlgVarsMainArea::dlgVarsMainArea(QWidget* pF) : QWidget(pF)
     // Now populate the widget - throw away stuff entered from form design
     // before substitute model was attached:
     comboBox_variable_value_type->clear();
-    comboBox_variable_value_type->insertItem(0, tr("Auto-Type"), -1); // LUA_TNONE
-    comboBox_variable_value_type->insertItem(1, tr("string"), 4);  // LUA_TSTRING
-    comboBox_variable_value_type->insertItem(2, tr("number"), 3);  // LUA_TNUMBER
-    comboBox_variable_value_type->insertItem(3, tr("boolean"), 3);  // LUA_TNUMBER
```

Comment:
Was this the cause of the original issue being addressed by this PR?

Reply:
Yes

Reply:
Ah, I started reviewing this without seeing your comment. I will approve the change - but do you think about the string to integer tests being suspect for zero?

Reply:
I think it's valid - will investigate it separately